src/libraries/System.Text.Json/tests/Serialization/ConstructorTests/ConstructorTests.Exceptions.cs

```diff
@@ -251,10 +337,11 @@ public Parameterized_ClassWithUnicodeProperty(int a\u0467)
         }
 
         [Fact]
-        [ActiveIssue("JsonElement needs to support Path")]
```

Comment:
This was previously fixed -- the issue wasn't with JsonElement but in logic setting the current property.